great_expectations/core/usage_statistics/usage_statistics.py

```diff
@@ -681,9 +681,10 @@ def _handle_expectation_suite_usage_statistics(
     else:
         payload = copy.deepcopy(interactive_mode.value)
 
-    assert (
-        expectation_suite or expectation_suite_name
+    assert not (
+        expectation_suite_name is None and expectation_suite is None
```

Comment:
This `assert()` statement wasn't working correctly. Adjusted to something a bit more verbose for readability.